themes/digital.gov/src/scss/new/shortcodes/_note.scss

```diff
@@ -137,3 +137,9 @@ li .note {
     @include u-font("serif", "lg");
   }
 }
+
+.disclaimer {
```

Comment:
If this is a new variant we should use BEM for clarity. 


So this would be: `dg-note--disclaimer`.